xpk.py

```diff
@@ -4378,12 +4357,14 @@ def cluster_create(args) -> None:
     if install_nccl_code != 0:
       xpk_exit(install_nccl_code)
 
-  xpk_print('Creating ConfigMap for cluster')
-  create_cluster_configmaps_code = create_cluster_configmaps(
-      args, system, tensorboard_config, autoprovisioning_config
-  )
-  if create_cluster_configmaps_code != 0:
-    xpk_exit(create_cluster_configmaps_code)
+  # Currently autoprovisioning is not supported with Pathways.
+  if not args.enable_pathways:
```

Comment:
This code is not autoprovisioning specific. It will create a configmap describing what accelerator types are used in the cluster.

Reply:
Thanks for the catch! I udpated the logic to disable autoprovisioning specific code path within this function.